src/index.html

```diff
@@ -1,12 +1,448 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/style.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <a
+        class="header__logo_link"
+        href="#home"
+      >
+        <img  src="./images/logo.png" alt="logo">
```

Comment:
![image](https://github.com/mate-academy/layout_catalog/assets/54860641/9ea7e73d-1ad9-466b-b853-598da44ed51b)

Reply:
Why did you ignore this comment?

Reply:
My fault I didn't notice